doc/site/en/guides/border-router/beaglebone-black.md

```diff
@@ -1,338 +1,204 @@
-Partition 1 has been deleted.
+sudo systemctl stop bonescript-autorun.service
+sudo systemctl stop bonescript.socket
+sudo systemctl stop bonescript.service
+sudo systemctl stop cloud9.socket
+sudo systemctl stop cloud9.service
+sudo systemctl stop nodered.service
+sudo systemctl disable bonescript-autorun.service
+sudo systemctl disable bonescript.socket
+sudo systemctl disable bonescript.service
+sudo systemctl disable cloud9.socket
+sudo systemctl disable cloud9.service
+sudo systemctl disable nodered.service
+sudo systemctl daemon-relaod
 ```
 
-Create the new partition, using a partition number of 1, the same starting
-sector of the previous partition (`8192` in this example), and a size that's
-1 GB less that the SD card size. For example, if using an 8 GB SD card,
-specify a size of `+7G`. For a 16 GB SD card, specify a size of `+15GB`.
+Disable advertising the Cloud9 IDE and NodeRED services with Avahi by deleting
+the service files:
 
 ```
-Command (m for help): n
-Partition type
-   p   primary (0 primary, 0 extended, 4 free)
-   e   extended (container for logical partitions)
-Select (default p): p
-Partition number (1-4, default 1): 1
-First sector (2048-15126527, default 2048): 8192
-Last sector, +sectors or +size{K,M,G,T,P} (8192-15126527, default 15126527): +7G
-
-Created a new partition 1 of type 'Linux' and of size 7 GiB.
+sudo rm /etc/avahi/services/*
 ```
 
-Newer versions of `fdisk` prompt you to erase the old disk `ext4`
-signature. **Do not erase this signature.** Otherwise, data is corrupted and
-the entire image becomes useless.
+The filesystem for the uSD BeagleBone image is limited to 4GB to fit on most
+uSD cards. Expand the partition to enable usage of the entire storage capacity.
 
 ```
-Partition #1 contains a ext4 signature.
-
-Do you want to remove the signature [Y]/No: n
+sudo /opt/scripts/tools/grow_partitions.sh
 ```
 
-Write the new partition table to disk and quit `fdisk`:
+You are encouraged to read that helper script to find out how the filesystem is
+expanded. You will have to reboot the BeagleBone and re-login to use this new
+filesystem definition.
 
 ```
-Command (m for help): w
-The partition table has been altered.
-Calling ioctl() to re-read partition table.
-Re-reading the partition table failed.: Device or resource busy
-
-The kernel still uses the old table. The new table will be used at the next
-reboot or after you run partprobe(8) or kpartx(8).
+sudo shutdown -r now
 ```
 
-### 3. Resize the file system
+This will close your SSH session.
 
-Use `resize2fs` to resize the image file system to the newly-expanded partition
-size. This tool expands or shrinks a file system.
-
-Reboot the BBB.
-
-Some images may throw an `fsck` error upon reboot. `fsck` runs
-automatically on boot and checks for file system consistency. If you get
-this error, ignore it and wait about 20 seconds for the login
-prompt to appear.
+Once logged back into the BeagleBone install `network-manager` with the command
```

Comment:
comma after BeagleBone

and I would make `network-manager` be Network Manager since you refer to it like that later on.